azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/configuration/CallCompositeConfiguration.kt

```diff
@@ -14,4 +15,6 @@ internal class CallCompositeConfiguration {
     var callConfig: CallConfiguration? = null
     var callCompositeLocalOptions: CallCompositeLocalOptions? = null
     val remoteParticipantsConfiguration: RemoteParticipantsConfiguration = RemoteParticipantsConfiguration()
+    lateinit var callScreenOrientation: CallCompositeSupportedScreenOrientation
```

Comment:
keep nullable callScreenOrientation as this represents Contoso provided configuration